test/LightStep.Tests/LightStep.Tests.csproj

```diff
@@ -13,4 +14,10 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\LightStep\LightStep.csproj" />
   </ItemGroup>
+  <PropertyGroup>
```

Comment:
What are the effects of these changes?

Reply:
Per above, I needed to sign the test assembly in order to make it a friend of the main assembly. In practice, it doesn't change anything except allowing me to refer to `internal` methods in `LightStep` in tests.